dials_data/definitions/MyD88_processed.yml

```diff
@@ -0,0 +1,13 @@
+name: MyD88 processed
+author: David Waterman (2021)
+license: CC-BY 4.0
+description: >
+  The first dataset from https://doi.org/10.15785/SBGRID/814, imported
+  and strong spots found with DIALS:
```

Comment:
Put the URL into a `url:` field, and then describe what dataset this is instead.

When you say 'found with DIALS' - what version of DIALS?